### ISSUE TITLE
New form design

### ISSUE THREAD

**Issue by sofiapoh** [saferTogether/saferTogether#66]
Our talented designer @nogainbar has redesigned the questionnaire for us so it looks great with mobile and desktop! Let us know what you think and also let us know when you have decided what other pages you will want to create for this user flow.

**Mobile**
![](https://files.gitter.im/nogainbar/U2vR/safer_together_mobile.png)

**Desktop**
![](https://files.gitter.im/nogainbar/6B6A/safer_together1.png)

**Comment by Nicky-Cummings** [comment]
I like it! Any other thoughts?

So from the user feedback it was clear that they'd like to be able to access all pages from one front page. The areas they wanted to access are..
1. Questionnaire - Need to agree on the name of this link i.e. do you know if you are at risk? Please provide your thoughts on this.
2. Resources - same as above what do you think would sound good for the user experience
3. Contact Us - Form to submit details
4. Get help now -  Contact details of services

If you think of better naming for any of the areas above please provide details.

Also, we'd like to enhance the landing page at the end of the questionnaire to make suggestions for what users should do next based on their risk.

A quick 999 call button would be good too.

Please let me know what you think on the above.

Thank you,
Nicky

**Comment by shigufta-khan** [comment]
In light what is going on at the moment are we going ahead with this app?

[cid:image002.png@01CF9C2D.898D6480]Shigufta Khan | Chief Executive Officer
Blackburn & Darwen District
Without Abuse
43 King Street, Blackburn, BB2 2DH
Tel: 01254 260465

[cid:image005.png@01CF99C5.2F71FC00]Web: www.bddwa.org.uk<http://www.bddwa.org.uk/>
Email: shigufta.khan@bddwa.org.uk<mailto:shigufta.khan@bddwa.org.uk>


Supported by Comic Relief, Henry Smith and Children In Need. BDDWA is a registered charity and exists to assist in the relief of people who have suffered or who are at risk of suffering domestic abuse, through the provision of support, advice and accommodation.

[safer together logo (2)][cid:image001.png@01CF6197.DA126BB0][cid:image004.png@01CF6198.067B2250]

Registered company 5834929 Registered charity 1116444.

**Comment by Nicky-Cummings** [comment]
Hi Shigufta,

I spoke to Amanda Tuesday and said to progress with this phase so maybe worth a discussion.

I'll give you a call..

**Comment by shigufta-khan** [comment]
I like the yes / no questions its quicker and easier for the user to complete also like the idea of the landing page with  information that is relevant and needed at the time. I will think of some questions and come back to you.

**Comment by Nicky-Cummings** [comment]
Fab! Thanks Shigufta... Also, we discussed maybe rewording or adding to the questions to include coercive control etc. so please let us know your thoughts..

**Comment by nogainbar** [comment]
hey @Nicky-Cummings,

glad you like the design :)
as you can see, the whole website revolves around a 'tabbing' system (i.e 'questionnaire', 'what to do', 'i need support' ect..)  in which users can change tabs and so then the central bit of the webpage changes accordingly. 

I have a few questions so that I can continue with the design of the other pages, but I also wanted to answer your questions- re the names for the 'tabs', i always think that the best naming in terms of user experience, is a personalised one, so for example, instead of 'resources' maybe inside a 'what to do' or 'I need support'.. a more 'human friendly' naming is always good.

but eventually, you will need to send me the names of the tabs you would like to include and also the copy that goes inside them as I cannot know what must be included in your website otherwise..

could you please clarify what you mean by 'we'd like to enhance the landing page at the end of the questionnaire to make suggestions for what users should do next based on their risk.' - do you mean the results page which you arrive at when you finish the questionaire? if so, i suggest the according to the result, there will be links that will take you directly to the relevant tab where you can find the relevant information for you according to the result of the questionnaire. how does that sounds to you?

I am currently out of the county but back on tuesday so can talk more (on the phone perhaps) then but in the meantime, you can write to me here and so both @sofiapoh and @bradreeder can see it too and implement it if needed. 

Thank you,
Noga

**Comment by Nicky-Cummings** [comment]
Hello Noga,

Thank you so much..

Sorry for the delay I work part time and have just popped in. I'm in tomorrow and can call you to answer the queries you've raised if you are free? Is their a good number to contact you on? I am on 07545 473177.

I do mean the results page when you complete the questionnaire.. I think the idea of links sounds great hopefully we can discuss this tomorrow.

Many thanks,
Nicky

**Comment by Nicky-Cummings** [comment]
Hi Everyone,

We have been discussing the questionnaire on the App and think it maybe good to reword the questions and maybe make it more appropriate users i.e. people with or without children, young people.

What do you think?

I'm going to have a look into it and will get back to you with some possible suggestions.

Many thanks.,
Nicky

**Comment by Nicky-Cummings** [comment]
Actually the Women's Aid questionnaire is great...

https://www.womensaid.org.uk/the-survivors-handbook/am-i-in-an-abusive-relationship/

Do you think we should use these instead.. ?

**Comment by shigufta-khan** [comment]
This is very good I like the wording

[cid:image002.png@01CF9C2D.898D6480]Shigufta Khan | Chief Executive Officer
Blackburn & Darwen District
Without Abuse
43 King Street, Blackburn, BB2 2DH
Tel: 01254 260465

[cid:image005.png@01CF99C5.2F71FC00]Web: www.bddwa.org.uk<http://www.bddwa.org.uk/>
Email: shigufta.khan@bddwa.org.uk<mailto:shigufta.khan@bddwa.org.uk>


Supported by Comic Relief, Henry Smith and Children In Need. BDDWA is a registered charity and exists to assist in the relief of people who have suffered or who are at risk of suffering domestic abuse, through the provision of support, advice and accommodation.

[safer together logo (2)][cid:image001.png@01CF6197.DA126BB0][cid:image004.png@01CF6198.067B2250]

Registered company 5834929 Registered charity 1116444.

**Comment by HR2005** [comment]
Hi Nicky,

I do like how the women’s aid questionnaire is worded.

Regards

Mehmona

Mehmona Arif
Project Worker

Humraaz
PO Box 427
Blackburn
BB1 5BN

Tel 01254 674312
[image1][safer together logo (2)]

This message and any files transmitted with it are confidential and intended solely for the individual(s) addressed. If you have received this e-mail in error then please note that disclosing, copying, distributing or from your system. Any views or opinions presented are solely those of the author and do not necessarily represent those of Humraaz. The organisation accepts no liability for any damage caused by any virus transmitted with this e-mail, so although it has been virus checked before transmission, the recipient should also check for the presence of viruses.  The confidentiality of this email and any subsequent reply cannot be guaranteed.

**Comment by Nicky-Cummings** [comment]
[Development Sprint Doc.docx](https://github.com/saferTogether/saferTogether/files/684429/Development.Sprint.Doc.docx)
Hi Everyone,

Happy New Year!

I've updated the copy for the design sprint next week. Please could you review and get back to me with any comments asap.

I need more resources so please send me anything you think would be useful for victims accessing the site.

Many thanks,
Nicky
